gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.51.14
+version=29.51.15
```

Comment:
well, it only affects the indis flow. If apps do gRPC migration but still read from ZK, they'll be fine. But we could bump the minor version to force an update since it's kind of significant to INDIS flow.